site/data/breakpoints.yml

```diff
@@ -1,35 +1,47 @@
-- breakpoint: xs
+- breakpoint: 2xs
   abbr: ""
-  name: X-Small
+  name: 2X-Small
```

Comment:
In https://unified-design-system.orange.com/472794e18/p/26a48e-grid/t/e076cd8936, the names are rather "2xs", "xs", etc. Maybe we can use the same. Otherwise we can keep Bootstrap naming, "Extra extra small", etc.

Reply:
At first I wanted to keep Bootstrap names, but "Extra extra extra large" is really clumsy (lourdingue 😃 ). 
Anyway, it has to be the same everywhere (see above)